dv/uvm/core_ibex/riscv_dv_extension/testlist.yaml

```diff
@@ -587,3 +587,13 @@
     +pmp_max_offset=00021000
     +boot_mode=u
   rtl_test: core_ibex_base_test
+
+- test: riscv_bitmanip_test
+  desc: >
+    Random instruction test with supported B extension instructions
+  iterations: 1
```

Comment:
nit: can you change this to 10 iterations?

Reply:
Yep, done.